app.js

```diff
@@ -134,61 +125,66 @@ if (authService && authService.initialize)
 else
 	initDB();
 
-
-function checkStatus(req, res){
-	res.sendStatus(200);
+function checkStatus(req, reply) {
+	reply.send("OK");
 }
 
-function login(req, res){
-	if (!initialized)
-     {
+function login(req, reply) {
+	if (!initialized) {
 		logger.info("please wait for db connection initialized then trigger again.");
 		initDB();
-		res.sendStatus(403);
-	}else
-		routes.login(req, res);
+		reply.code(403).send('Forbidden');
+	} else {
+		routes.login(req, reply);
+	}
 }
 
-function logout(req, res){
+function logout(req, reply){
 	if (!initialized)
      {
 		logger.info("please wait for db connection initialized then trigger again.");
 		initDB();
-		res.sendStatus(400);
+		reply.code(400).send('Bad request');
 	}else
-		routes.logout(req, res);
+		routes.logout(req, reply);
 }
 
 
-function startLoadDatabase(req, res){
+function startLoadDatabase(req, reply){
 	if (!initialized)
      	{
 		logger.info("please wait for db connection initialized then trigger again.");
 		initDB();
-		res.sendStatus(400);
+		reply.code(400).send('Bad request');
 	}else
-		loader.startLoadDatabase(req, res);
+		loader.startLoadDatabase(req, reply);
 }
 
 function initDB(){
-    if (initialized ) return;
-		routes.initializeDatabaseConnections(function(error) {
-	if (error) {
-		logger.info('Error connecting to database - exiting process: '+ error);
-		// Do not stop the process for debug in container service
-		//process.exit(1); 
-	}else
-	      initialized =true;
-
-	logger.info("Initialized database connections");
-	startServer();
+	if (initialized ) return;
+	routes.initializeDatabaseConnections(function(error) {
```

Comment:
this is not a good pattern. The routes should not have this type of init phase, but rather you should have a plugin that provides the database connection to them. You can also use fastify-mongodb.